pyuvsim/tests/test_telescope.py

```diff
@@ -181,6 +181,23 @@ def test_beamlist_errors(beam_objs):
     with pytest.raises(ValueError, match='Invalid file path'):
         beamlist.append(beam_path)
 
+    # test error on beams with different x_orientation
+    newbeams[0].x_orientation = None
+    with pytest.raises(
+        ValueError, match="UVBeam x_orientations do not match among beams in list."
```

Comment:
I'm happy not to, but it might be worth considering using a custom error, like `BeamListConsistencyError`. It may be useful later to be able to catch such errors in certain contexts.

Reply:
Ok. I don't have strong feelings on this either way. I think we should talk about how this PR and PR #360 interact.